.github/workflows/releaseChart.yaml

```diff
@@ -1,8 +1,15 @@
 name: Release newrelic prometheus configurator chart
+# This workflow manually triggers the automated process of release based on the changelog.
+# Is recommended to run `make release-changelog` before to see an example of what is going to be generated.
+
+# IMPORTANT No matter the branch selected when triggered, the workflow will always execute on 'main' branch.
+
 on:
-  push:
-    branches:
-      - main
+  workflow_dispatch:
```

Comment:
would make sense to add some optional parameters that would skip the rt steps for any time we need to release without this tool?

Reply:
discussed it offline, no for now